pkg/apis/sources/v1/container_conversion_test.go

```diff
@@ -25,10 +25,10 @@ func TestContainerSourceConversionBadType(t *testing.T) {
 	good, bad := &ContainerSource{}, &testObject{}
 
 	if err := good.ConvertTo(context.Background(), bad); err == nil {
-		t.Errorf("ConvertTo() = %#v, wanted error", bad)
+		t.Error("ConvertTo() = nil, wanted error")
 	}
 
 	if err := good.ConvertFrom(context.Background(), bad); err == nil {
-		t.Errorf("ConvertFrom() = %#v, wanted error", good)
+		t.Errorf("ConvertFrom() = nil, wanted error")
```

Comment:
Should this be `t.Error` instead? Same comment applies to the other similar changes below.

Reply:
Yeah somehow slipped.
Fixed.